qt/python/mantidqt/widgets/plotconfigdialog/presenter.py

```diff
@@ -93,3 +93,18 @@ def open_help_window(self):
             InterfaceManager().showHelpPage(HELP_URL + '#image-plots')
         else:
             InterfaceManager().showHelpPage(HELP_URL + '#figureoptionsgear-png-ptions-menu')
+
+    def forget_tab_from_presenter(self, tab_presenter):
+        """Given the presenter of a tab, forgets the tab's presenter and view
+        by setting them to none, for example when the tab view is closed."""
+        for index, view_name_pair in enumerate(self.tab_widget_views):
+            if view_name_pair:
```

Comment:
Shouldn't this always be true, unless the data structure gets out of sync somehow?

Reply:
It's a bit funny - it's an array of size four to reflect the four possible tabs that can exist, but not all of them have to exist so they're set to none. I think it's so that it's easier to maintain which order the tabs should appear?

Reply:
Ahh that makes sense to keep the ordering too, otherwise the tabs would swap around depending on their creation order

Thanks for the cleanup